src/test/proc_server/biz_set_test.go

```diff
@@ -0,0 +1,408 @@
+		input := map[string]interface{}{
+			"bk_biz_id": resBizId,
+			"host_info": map[string]interface{}{
+				"1": map[string]interface{}{
+					"bk_host_innerip": "127.0.0.3",
+					"bk_asset_id":     "addhost_api_asset_1",
+					"bk_cloud_id":     0,
+				},
+				"2": map[string]interface{}{
+					"bk_host_innerip": "127.0.0.4",
+					"bk_asset_id":     "addhost_api_asset_2",
+					"bk_cloud_id":     0,
+				},
+			},
+		}
+		rsp, err := hostServerClient.AddHost(context.Background(), header, input)
+		util.RegisterResponseWithRid(err, header)
+		Expect(err).NotTo(HaveOccurred())
+		Expect(rsp.Result).To(Equal(true))
+
+		searchInput := &metadata.HostCommonSearch{
+			AppID: resBizId,
+		}
+		resp, err := hostServerClient.SearchHostWithBiz(context.Background(), header, searchInput)
+		util.RegisterResponseWithRid(err, header)
+		Expect(err).NotTo(HaveOccurred())
+		Expect(resp.Result).To(Equal(true))
+		Expect(resp.Data.Count).To(Equal(2))
+		hostID1, err = commonutil.GetInt64ByInterface(resp.Data.Info[0]["host"].(map[string]interface{})["bk_host_id"])
+		Expect(err).NotTo(HaveOccurred())
+		hostID2, err = commonutil.GetInt64ByInterface(resp.Data.Info[1]["host"].(map[string]interface{})["bk_host_id"])
+		Expect(err).NotTo(HaveOccurred())
+
+		moduleResult := make(map[string]interface{})
+		searchModuleCond := map[string]interface{}{
+			"bk_biz_id":      bizID1,
+			"bk_module_name": common.DefaultResModuleName,
+		}
+		err = test.GetDB().Table(common.BKTableNameBaseModule).Find(searchModuleCond).Fields("bk_module_id").One(context.Background(),
+			&moduleResult)
+		Expect(err).NotTo(HaveOccurred())
+		resModuleID, err := commonutil.GetInt64ByInterface(moduleResult["bk_module_id"])
+
+		Expect(err).NotTo(HaveOccurred())
+		transferInput := &metadata.TransferHostAcrossBusinessParameter{
+			SrcAppID:    resBizId,
+			DstAppID:    bizID1,
+			HostID:      []int64{hostID1, hostID2},
+			DstModuleID: resModuleID,
+		}
+		transferErr := hostServerClient.TransferHostAcrossBusiness(context.Background(), header, transferInput)
+		Expect(transferErr).NotTo(HaveOccurred())
+
+		transInput := map[string]interface{}{
+			"bk_biz_id": bizID1,
+			"bk_host_id": []int64{
+				hostID1,
+			},
+			"bk_module_id": []int64{
+				moduleID1,
+			},
+			"is_increment": true,
+		}
+		rsp, rawErr := hostServerClient.TransferHostModule(context.Background(), header, transInput)
+		util.RegisterResponseWithRid(rsp, header)
+		Expect(rawErr).NotTo(HaveOccurred())
+		Expect(rsp.Result).To(Equal(true))
+
+		transInput2 := map[string]interface{}{
+			"bk_biz_id": bizID1,
+			"bk_host_id": []int64{
+				hostID2,
+			},
+			"bk_module_id": []int64{
+				moduleID2,
+			},
+			"is_increment": true,
+		}
+		rsp, rawErr = hostServerClient.TransferHostModule(context.Background(), header, transInput2)
+		util.RegisterResponseWithRid(rsp, header)
+		Expect(rawErr).NotTo(HaveOccurred())
+		Expect(rsp.Result).To(Equal(true))
+	})
+	It("creat service instance", func() {
```

Comment:
```suggestion
	It("create service instance", func() {
```
然后在另一个业务也建一个服务实例，不应该被查到